src/input/metadata/mod.rs

```diff
@@ -1,9 +1,11 @@
+//! Metadata about outputs - currently only [`ytdl::Output`] is publicly exported.
+
 use crate::error::JsonError;
 use std::time::Duration;
 use symphonia_core::{meta::Metadata as ContainerMetadata, probe::ProbedMetadata};
 
 pub(crate) mod ffprobe;
-pub(crate) mod ytdl;
+pub mod ytdl;
```

Comment:
I'd be tempted to recast this as:

```suggestion
mod ytdl;
pub use ytdl::Output as YoutubeDlMetadata;
```

Will need some fixup in `songbird\src\input\sources\ytdl.rs`, but I think that makes it easier to reach as a user.